### ISSUE TITLE
Choice of renderings for `diff` and `partialdiff`

### ISSUE THREAD

**Issue by christianp** [mathjax/ctop#8]
http://www.w3.org/Math/testsuite/build/main/Content/Calculus/degree/rec-degree1-form.xhtml
http://www.w3.org/Math/testsuite/build/main/Content/Calculus/diff/rec-diff2-form.xhtml

**Comment by pkra** [comment]
I don't really know enough about diff. equations to make a call. Will different people have different preferences? I prefer the image in the first link to ctop's current rendering (which resolves things to `d^n+m`) But that's much like the discussion about `7+ (-1)`...

**Comment by christianp** [comment]
On reflection, I also prefer `d^n+m`. 
For the second one, I've just written code to put `f(x)` to the right of the fraction in `d/dx f(x)`, because I don't like having too much stuff on the top.

**Comment by pkra** [comment]
IIUC, this means ctop.js now produces something like `\frac{d^{n+m}}{d x^n d y^m }  f(x)`?

**Comment by christianp** [comment]
Yeah, but not for partial differentiation yet because I'm still working on that.

**Comment by pkra** [comment]
sounds good.

**Comment by christianp** [comment]
I don't think I prefer the multiple-fractions version of partialdiff enough to implement it.

**Comment by pkra** [comment]
I think that's fine for this iteration.
